doc/source/reference/maskedarray.baseclass.rst

```diff
@@ -35,12 +35,19 @@ defines several constants.
    is not needed. It is represented internally as ``np.False_``.
 
 
-.. data:: masked_print_options
+.. data:: masked_print_option
 
    String used in lieu of missing data when a masked array is printed.
    By default, this string is ``'--'``.
 
 
+   
+   Add ``set_display()`` to change the default string.
+   Example usage: ``numpy.ma.masked_print_option.set_display('·')`` 
+   replaces missing data with ``'.'``.
+   
+
+
```

Comment:
It might be more illustrative to use a code example, e.g. 

```suggestion
   Use ``set_display()`` to change the default string::
   
       >>> a = ma.array([1, 2, 3], mask=[0, 1, 0])
       >>> ma.masked_print_option.set_display("X")
       >>> a
       masked_array(data=[1, X, 3],                                             
                    mask=[False,  True, False],                                    
              fill_value=999999)
```

Reply:
gotcha! Ill make the suggested changes thank you!

Reply:
<img width="1102" alt="スクリーンショット 2022-07-20 午後11 17 49" src="https://user-images.githubusercontent.com/105326591/180005408-b7688984-1046-43eb-968a-f4b5335a1214.png">

I guess the changes have affected other files...@rossbar

Reply:
I have tweaked the usage description text, but removed the code example part

Reply:
@MikiPWata good point, `set_display` sets the value globally, so this will affect all subsequent code blocks in the document. You'd either need to reset it after the example (i.e. add `ma.masked_print_option.set_display("--")  # Reset default` to the end of the example) or forgo the example entirely. I'm fine with option 2, though documenting that this sets a *global* config value might also be worthwhile.

I'm inclined not to get too in the weeds here though - we can always consider additional updates in subsequent PRs. The proposed change is an obvious improvement :+1: